python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -109,6 +109,23 @@ def run_mtdome() -> None:
     asyncio.run(MTDomeCsc.amain(index=None))
 
 
+def support_dds_item(command_name: str) -> bool:
```

Comment:
Consider making a utils module for this.

Reply:
Done